_includes/modern-appendix.md

```diff
@@ -813,6 +813,14 @@ Indicates that a {% message MODE %} command affecting a user contained a `MODE`
 
 Indicates that a {% message MODE %} command affecting a user failed because they were trying to set or view modes for other users. The text used in the last param of this message varies, for instance when trying to view modes for another user, a server may send: `"Can't view modes for other users"`.
 
+{% numericheader ERR_HELPNOTFOUND %}
+
+      "<client> <subject> :No help available on this topic"
+
+Indicates that a {% message HELP %} command requested help on a subject the server does not know about.
+
+The `<subject>` MUST be the one requested by the client, up to casing; unless it would be an invalid parameter, in which case it MUST be `*`.
```

Comment:
```suggestion
The `<subject>` MUST be the one requested by the client, but may be casefolded; unless it would be an invalid parameter, in which case it MUST be `*`.
```